documentation/auto-discovery/topics/advanced-discovery-topics.rst

```diff
@@ -0,0 +1,101 @@
+.. include:: /auto-discovery/media.rst
+.. _Advanced-discovery-topics-doc:
+
+*************************
+Advanced discovery topics
+*************************
+
+====================
+Obtaining SNMP walks
+====================
+
+Hyperview uses SNMP walks to enhance device definitions, to model and support devices that are discoverable with the SNMP protocol. The **snmpwalk file** is used to simulate the device and to test definitions.
+
+It is recommended to install the applicable net-snmp package alongside the (Linux) Data Collector software. If that is not possible it can be installed on any machine that has a network line of sight to the target device.
+
+Linux
+-----
+
+The `net-snmp <http://www.net-snmp.org/>`_ utilities are available on all supported Linux distributions.
+
+On Debian-based distributions:
+
+.. code::
+
+    sudo apt install snmp
+
+On RedHat-based distributions:
+
+.. code::
+
+    sudo dnf install net-snmp-utils
+
+Windows
+-------
+
+There are multiple options.
+
+1. Use net-snmp using a UNIX environment simulator such as `Cygwin <https://www.cygwin.com/>`_ or `WSL <https://docs.microsoft.com/en-us/windows/wsl/>`_.
+
+2. Use a Linux Virtual Machine
+
+macOS
+-----
+
+You can install net-snmp on macOS using `Homebrew <https://brew.sh/>`_
+
+.. code::
+
+    brew install net-snmp
+
+Once the application is installed, the **snmpwalk** command can be used to obtain a full walk of a device. The amount of time it takes to perform the walk can vary from device to device and depends on external factors such as network speed and target device CPU load. Generally, expect the process to take a few minutes unless the device has a large amount of information such as a large network switch, in which case, it may take longer.
```

Comment:
If you are connecting to a Linux system via putty/ssh then there is a chance that you could disconnect before the snmpwalk operation has completed if keepalive is not enabled or working and the walk takes more than a couple minutes. This would end the snmpwalk job. We may want to mention that you can enable ssh keepalive or use screen, tmux, or nohup so that the operation will complete even if the connection has been terminated.

This may be more suitable for a support forum article though. I'll let you make the call.

Reply:
@hvjamieconner is right.  We probably should do a support article and reference it here.